### ISSUE TITLE
Default to the spec?

### ISSUE THREAD

**Issue by KyleAMathews** [zippyui/react-flex#1]
I was surprised to find that `alignItems` defaults to `center` not `spread` as [the spec specifies](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items). Any reason for this?

**Comment by radubrehar** [comment]
You mean `stretch`?

**Comment by KyleAMathews** [comment]
Yes

**Comment by radubrehar** [comment]
I generally found that I'm using alignItems center more often. We could consider reverting to the default value with a major version bump

**Comment by KyleAMathews** [comment]
Yeah. It's just I was copying code from an example online and it took me a
while to discover why it wasn't working correctly.

**Comment by Zephir77167** [comment]
:+1:
